Sources/Clappr/Classes/Plugin/Playback/AVFoundationPlayback.swift

```diff
@@ -308,6 +294,26 @@ open class AVFoundationPlayback: Playback {
         }
     }
 
+    open override func render() {
+        if self.startAt != 0.0 && self.playbackType == .vod {
+            self.seek(self.startAt)
```

Comment:
The seek should be made right before the video starts playing for the first time. Are we sure this gonna work calling this method in the render?